src/components/Layout/HomeContent.js

```diff
@@ -131,100 +131,100 @@ export function HomeContent() {
               type="primary"
               size="lg"
               className="w-full sm:w-auto justify-center"
-              label="Learn React">
-              Learn React
+              label="Aprenda React">
+              Aprenda React
             </ButtonLink>
             <ButtonLink
               href={'/reference/react'}
               type="secondary"
               size="lg"
               className="w-full sm:w-auto justify-center"
-              label="API Reference">
-              API Reference
+              label="API Referência">
+              API Referência
             </ButtonLink>
           </div>
         </div>
 
         <Section background="left-card">
           <Center>
-            <Header>Create user interfaces from components</Header>
+            <Header>Crie interfaces de usuários a partir de componentes</Header>
             <Para>
-              React lets you build user interfaces out of individual pieces
-              called components. Create your own React components like{' '}
-              <Code>Thumbnail</Code>, <Code>LikeButton</Code>, and{' '}
-              <Code>Video</Code>. Then combine them into entire screens, pages,
-              and apps.
+              O React permite que você crie interfaces de usuário a partir de
+              peças individuais chamadas de componentes. Crie seus próprios
+              componentes React como <Code>Thumbnail</Code>,{' '}
+              <Code>LikeButton</Code>, e <Code>Video</Code>. Em seguida,
+              combine-os em telas interas, páginas, e aplicativos.
             </Para>
           </Center>
           <FullBleed>
             <Example1 />
           </FullBleed>
           <Center>
             <Para>
-              Whether you work on your own or with thousands of other
-              developers, using React feels the same. It is designed to let you
-              seamlessly combine components written by independent people,
-              teams, and organizations.
+              Quer você trabalhe sozinho ou com milhares de outros
+              desenvolvedores, usar o React parece o mesmo. Ele foi projetado
+              para permitir que você combine componentes escritos por pessoas,
+              equipes e organizações independentes.
             </Para>
           </Center>
         </Section>
 
         <Section background="right-card">
           <Center>
-            <Header>Write components with code and markup</Header>
+            <Header>Escrever componentes com código e marcação</Header>
             <Para>
-              React components are JavaScript functions. Want to show some
-              content conditionally? Use an <Code>if</Code> statement.
-              Displaying a list? Try array <Code>map()</Code>. Learning React is
-              learning programming.
+              Componentes React são funções JavaScript. Quer mostrar alguns
+              conteúdos condicionalmente? Use uma instrução <Code>if</Code>.
```

Comment:
O que você acha de alterar de `Use uma instrução` para `Utilize`/`Use`?
```suggestion
              conteúdos condicionalmente? Utilize <Code>if</Code>.
```

Não sei se há a necessidade de referenciar `instrução`.